staging/src/k8s.io/kube-scheduler/config/v1/types.go

```diff
@@ -96,6 +96,31 @@ type KubeSchedulerConfiguration struct {
 	// failover with the benefit of lower memory overhead while waiting to become leader.
 	// Defaults to false.
 	DelayCacheUntilActive bool `json:"delayCacheUntilActive,omitempty"`
+
+	// Metric defines the configuration of metrics.
+	// +optional
+	Metric KubeSchedulerMetricConfiguration `json:"metric,omitempty"`
+}
+
+type KubeSchedulerMetricConfiguration struct {
+	// Percentage of plugin metrics to sample.
+	// A value of 100 means all metrics are recorded; lower values enable probabilistic sampling.
+	// Defaults to 10
+	PluginMetricsSamplePercent *int64 `json:"pluginMetricsSamplePercent,omitempty"`
+
+	// Buffer size for the MetricsAsyncRecorder.
```

Comment:
Let's not use `MetricsAsyncRecorder`. That's too internal for users

```suggestion
	// MetricsAsyncRecorderBufferSize defines the buffer size for the MetricsAsyncRecorder, i.e., how many metric values the buffer can hold.
```